main.go

```diff
@@ -212,9 +230,9 @@ func doMain() error {
 	// Here we can't use buildutil.ForEachPackage here since it only considers
 	// src dirs and this tool should be able to run against a golang source dir.
 	forEachPackage(&ctxt, func(path string, err error) {
-		if path == "" {
-			return
-		}
+		// if path == "" {
+		// 	return
+		// }
```

Comment:
Why was this changed? Please remove it instead of just uncommenting it.

Reply:
it's not empty package, but "empty relative path" to path specified as argument... so this allows if `/home/user/go/src/github.com/acroca/go-symbols` specified as path for `go-symbols` list symbols in this folder and subfolders, not only in subfolders